superset/assets/src/SqlLab/actions.js

```diff
@@ -34,6 +34,7 @@ export const SET_DATABASES = 'SET_DATABASES';
 export const SET_ACTIVE_QUERY_EDITOR = 'SET_ACTIVE_QUERY_EDITOR';
 export const SET_ACTIVE_SOUTHPANE_TAB = 'SET_ACTIVE_SOUTHPANE_TAB';
 export const REFRESH_QUERIES = 'REFRESH_QUERIES';
+export const USER_OFFLINE = 'USER_OFFLINE';
```

Comment:
Rename this to `SET_USER_OFFLINE` to make verb like other action names.

Reply:
very good catch!